frontend/pubspec.yaml

```diff
@@ -92,6 +92,7 @@ flutter:
     - assets/images/events/
     - assets/files/
     - .env
+    - assets/images/events/gameplay.jpg
```

Comment:
You should be able to remove this, since `assets/images/events` is already set above.

Reply:
Thank you for pointing out the redundant line. 
I appreciate your attention to detail, I will definitely address it in my next set of code changes. : )